drivers/scheduler/k8s/specs/fio-storagepool/fio-config-map.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fio-job-config
+data:
+    fio.job: |
+        [global]
+        name=fio-write
```

Comment:
any specific why we are creating new spec?

Reply:
mostly to simplify what the spec does, which is create a fio vol and write to it.